lib/slack.js

```diff
@@ -0,0 +1,60 @@
+const https = require("https");
+
+const slackWebhookHost = "hooks.slack.com";
+
+const channelConfig = {
```

Comment:
I set this in lambda so it's always SLACKWEBHOOKDEVCHANNEL, it's just set to the appropriate value depending on context

Reply:
Per our convo, I think we're all set here. SLACKWEBHOOKBOTCHANNEL is correct SLACKWEBHOOKBOTTESTINGCHANNEL is just for DEVELOPMENT.